app/SettledExpense.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class SettledExpense extends Model
```

Comment:
Same as above, treat settled expenses as expenses, just set a column for settled.